test/test-network/conf/25-l2tp-udp.netdev

```diff
@@ -12,8 +12,8 @@ Local=static@test1
 Remote=192.168.30.101
 EncapsulationType=udp
 UDPCheckSum=true
-UDP6CheckSumRx=true
-UDP6CheckSumTx=true
+UDP6ChecksumRx=true
```

Comment:
Maybe I shouldn't change a test for backwards compatibility reasons.

Reply:
Please use the new names. Old names are checked in fuzzers.